core/tests/webdriverio_utils/ProfilePage.js

```diff
@@ -14,29 +14,24 @@
 
 /**
  * @fileoverview Page object for the profile page, for use in
- * Protractor tests.
+ * WebdriverIO tests.
  */
+
 var waitFor = require('./waitFor.js');
 
 var ProfilePage = function() {
-  var currUserProfilePhoto = element(
-    by.css('.protractor-test-profile-current-user-photo'));
-  var otherUserProfilePhoto = element(
-    by.css('.protractor-test-profile-other-user-photo'));
-  var bio = element(
-    by.css('.protractor-test-profile-bio'));
-  var interests = element.all(
-    by.css('.protractor-test-profile-interest'));
-  var interestPlaceholder = element(
-    by.css('.protractor-test-profile-no-interest'));
-  var allExplorationCardElements = element.all(
-    by.css('.protractor-test-exploration-dashboard-card'));
-  var createdExplorationStat = element.all(
-    by.css('.protractor-test-profile-created-stat'));
-  var cardTitleCss = by.css('.protractor-test-exp-summary-tile-title');
+  var allExplorationCardElement = $('.e2e-test-exploration-dashboard-card');
+  var bio = $('.e2e-test-profile-bio');
+  var createdExplorationStat = $('.e2e-test-profile-created-stat');
+  var currUserProfilePhoto = $('.e2e-test-profile-current-user-photo');
+  var interestPlaceholder = $('.e2e-test-profile-no-interest');
+  var interestsSelector = function() {
+    return $$('.e2e-test-profile-interest');
```

Comment:
Instead of creating a async function I am using await where we are calling this function PTAL at line no 58, because async function sometimes returns an unresolved promise which can fail the test.